src/styles/utils/_extends.scss

```diff
@@ -1,4 +1,41 @@
 %h1 {
-  font-family: "Roboto", sans-serif;
-  font-weight: 400;
+  font-family: "Poppins", sans-serif;
+  font-weight: 700;
+  color: #fff;
+}
+
+%h2 {
+  font-family: "Poppins", sans-serif;
```

Comment:
is it really necessary to set font-family so many times?
you could provide it for the entire body, right?